student.rb

```diff
@@ -1,7 +1,6 @@
 require_relative 'person'
 
 class Student < Person
-
   def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
     super(id, age, parent_permission, name)
     @classroom = classroom
```

Comment:
- [ ] As suggested by the previous reviewer kindly remove id from initialize and super because your person class doesn't take id as input from the user and your app is supposed to generate the id automatically :+1: